packages/ckeditor5-build-helpjuice-v2/src/plugins/image/customimage.js

```diff
@@ -0,0 +1,56 @@
+/**
+ * @license Copyright (c) 2003-2023, CKSource Holding sp. z o.o. All rights reserved.
+ * For licensing, see LICENSE.md or https://ckeditor.com/legal/ckeditor-oss-license
+ */
+
+import Plugin from '@ckeditor/ckeditor5-core/src/plugin';
+import ImageInline from '@ckeditor/ckeditor5-image/src/imageinline';
+import ImageBlock from '@ckeditor/ckeditor5-image/src/imageblock';
+
+export default class MyImagePlugin extends Plugin {
```

Comment:
Can you change the name to something different? Maybe `CustomImageAttributesPlugin` (in this case you can also change the filename)?

Reply:
Done